remotes/handlers.go

```diff
@@ -56,11 +56,22 @@ func FetchHandler(ingester content.Ingester, fetcher Fetcher) images.HandlerFunc
 func fetch(ctx context.Context, ingester content.Ingester, fetcher Fetcher, desc ocispec.Descriptor) error {
 	log.G(ctx).Debug("fetch")
 	ref := MakeRefKey(ctx, desc)
+
+	cw, err := ingester.Writer(ctx, ref, desc.Size, desc.Digest)
```

Comment:
@dmcgowan Initially posted on this PR cause I thought we should update this PR to let `Writer` return `ErrExists` error. But I agree what you said https://github.com/containerd/containerd/pull/885#discussion_r118132915 makes more sense.

Sure. Will file an issue.